utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
+		position['latitude'] = tonumber(s.latitude) --owm
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Hi

Am 14.02.16 um 18:03 schrieb Sven Roederer:
>> @@ -205,6 +205,15 @@ function showmac(mac)
>>  	return mac
>>  end
>>  
>> +function get_position()
>> +	local position = {}
>> +	uci:foreach("system", "system", function(s) --owm
>> +		position['latitude'] = tonumber(s.latitude) --owm
> 
> cleaned / moved them back to original positions. but there are many of these "--owm" comments.
> 
> looks like they are marking the datafields of the OWM-json
> @stargieg what do you think
Yes. This fields are necessary for owm db

-----BEGIN PGP SIGNATURE-----
Version: GnuPG/MacGPG2 v2
Comment: GPGTools - http://gpgtools.org

iEYEARECAAYFAlbAvxgACgkQr9m0OkMZoqTB7ACgpizUcmtWMHLLQx4kw8dOUQ6j
MB4An1+tDOtzJktBHOewUU3w2gP/rNXI
=7U1L
-----END PGP SIGNATURE-----